### PR TITLE
Add email and passwords validation on the server and client sides

### PR DESCRIPTION
**Pull request by sharok89**

### CODE REVIEW

File: client/js/localization/words/en.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ var localization = {
     YES: 'yes',
     OK: 'ok',
     CONFIRM: 'confirm',
-    NO_DATE: 'no date'
+    NO_DATE: 'no date',
+    VALIDATION_WRONG_EMAIL: 'The given email is incorrect.',
+    VALIDATION_EMPTY_PASSWORD: 'Password cannot be empty.',
+    VALIDATION_WRONG_CONFIRM_PASSWORD: 'Password does not match the confirm password'
 };
```
Comment (atomiomi) [inline client/js/localization/words/en.js]: Three fields above should be moved to the `sentences` module

File: client/js/pages/login.jsx [hunk 2]
```diff
@@ -31,6 +32,20 @@ var Login = React.createClass({
         })
     },
 
+    validateForm: function (email, password) {
+        if (!validator.checkEmail(email)) {
```
Comment (atomiomi) [inline client/js/pages/login.jsx]: Does it make sense to check email in login form?  In any case user gets error about incorrect email or password

**Comment by atomiomi** [comment]
Looks good, I will merge it after you make fixes.

File: client/js/localization/words/en.js (hunk 1 above)
Comment (sharok89) [inline client/js/localization/words/en.js]: "No date" is not one word also :). I think we have to reconsider how to store strings. I don't see the pros of separating like that.

File: client/js/pages/login.jsx (hunk 2 above)
Comment (sharok89) [inline client/js/pages/login.jsx]: Yes, it decrease number of request to the server.

File: client/js/localization/words/en.js (hunk 1 above)
Comment (atomiomi) [inline client/js/localization/words/en.js]: Yeh I know, but now we have such implementation and they should be separated sebsequently it be more convenient to reconstruct this logic

File: client/js/pages/login.jsx (hunk 2 above)
Comment (atomiomi) [inline client/js/pages/login.jsx]: Ok

**Pull request merged**